configs/katello_37.yaml

```diff
@@ -0,0 +1,190 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.6.0:
+    :redmine_release_id: 284
+  :3.6.1:
+    :redmine_release_id: 352
+:releases:
+  :3.7.0:
+    :redmine_release_id: 338
+:code_name: Blonde Ale
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.7
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :katello-selinux:
+    :branch: 3.0.3
+    :repo: https://github.com/Katello/katello-selinux.git
+  :katello-installer:
+    :branch: KATELLO-3.7
+    :repo: https://github.com/Katello/katello-installer.git
+    :version_branch: true
+  :hammer-cli-katello:
+    :branch: 0.12.1
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :foreman-packaging:
+    :branch: rpm/1.17
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :katello-host-tools:
+    :branch: 3.2.0
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :bastion:
+    :branch: 6.1.10
+    :repo: https://github.com/Katello/bastion
+  :puppet-pulp:
+    :branch: 5.4.0
+    :repo: https://github.com/Katello/puppet-pulp
+  :puppet-foreman_proxy_content:
+    :branch: 7.0.0
+    :repo: https://github.com/Katello/puppet-foreman_proxy_content
+  :puppet-katello:
+    :branch: 7.2.0
+    :repo: https://github.com/Katello/puppet-katello
+  :puppet-candlepin:
+    :branch: 5.1.0
+    :repo: https://github.com/Katello/puppet-candlepin
+  :puppet-certs:
+    :branch: 4.0.1
+    :repo: https://github.com/Katello/puppet-certs
+
+:gpg_key: '0D3F9F9F'
+:mash_scripts:
+ - katello-3.7-thirdparty-candlepin-rhel7
+ - katello-3.7-thirdparty-pulp-rhel7
```

Comment:
@jlsherrill Do we still need a mash script for this in light of our switch to using Pulp repos directly?